src/Constants.js

```diff
@@ -49,11 +49,24 @@ export const UPDATE_EXPERIENCE_ENDPOINT =
 export const UPDATE_CERTIFICATE_ENDPOINT =
   "/api/profiles/:profile_id/certificates/:certificate_id";
 
+export const DELETE_PROFILE_ENDPOINT = "/api/profiles/:profile_id";
+export const DELETE_ACHIEVEMENT_ENDPOINT =
+  "/api/profiles/:profile_id/achievement/:achievement_id";
```

Comment:
wrong API path - 
achievement it should be plural, achievements, projects, experiences etc

Reply:
Done!